training/README.md

```diff
@@ -0,0 +1,41 @@
+Training in Digital Publishing
+===========================
+
+:warning: **This section is very much a work in progress. Links may not resolve, and content linked to may be unfinished** 
+
+When joining Digital Publishing, there can be a lot to learn! We want to give you the benefit of all our previous learning, so we've tried to gather it in one place.
+
+We have collected resources that give you an idea of [structured learning](#onboarding-workshops) you'll be undertaking over your first few weeks as well as [training materials](#further-training-areas) the team have found useful over the years.
+
+If you come across a resource that really helped you understand some technology we use, or how we work, please do add a link to it in the relevant document or start a new doc if the topic is not yet covered.
+
+
+Onboarding workshops
+----------------------------
+* [Cloud technologies and Infrastructure](INFRASTRUCTURE.md)
+* [Core Services](CORE.md)
+* [Microservices & Customise my Data](MICRO.md)
+* [Developer Processes](WORKFLOW.md)
+* [Intro to Go](GO.md)
+* [Platform Services](PLATFORM.md)
+
+* [Support](SUPPORT.md)
+
+
+Further training areas
+-----------------------
+
+* [Architectural Concepts](ARCHITECTURE.md)
```

Comment:
I would be tempted to move this up under onboarding. It feels like everyone should understand what architecture is, and how to understand our approach and documentation.